src/main/groovy/edu/oregonstate/mist/directoryapi/DirectoryEntityDAO.groovy

```diff
@@ -0,0 +1,86 @@
+package edu.oregonstate.mist.directoryapi
+
+import org.ldaptive.DefaultConnectionFactory
+import org.ldaptive.Connection
+import org.ldaptive.LdapEntry
+import org.ldaptive.Response
+import org.ldaptive.SearchOperation
+import org.ldaptive.SearchRequest
+import org.ldaptive.SearchResult
+
+/**
+ * Directory entity data access object.
+ */
+class DirectoryEntityDAO {
+    // FIXME: define properties in external file?
+    private final String LDAP_URL = 'ldap://client-ldap.onid.orst.edu'
```

Comment:
Yes, both of these strings should be coming from a configuration file. We don't want to be hard coding them like this.